charts/aws-vpc-cni/values.yaml

```diff
@@ -54,6 +54,7 @@ env:
   DISABLE_NETWORK_RESOURCE_PROVISIONING: "false"
   ENABLE_IPv4: "true"
   ENABLE_IPv6: "false"
+  ENABLE_NFTABLES: "false"
```

Comment:
Since it is false by default, let's not add it to values.yaml or manifest.

Reply:
Doesn't it need to be here to be configurable?

Reply:
Yeah we don't need to. Previously we kept adding and actually it isn't needed. Since if configured the value will be picked from the env or else we have default as false.